### PR TITLE
Use Kepler to retrieve product usage metrics

### PR DESCRIPTION
**Pull request by alexandrebouthinon**
## What does this PR do ?

As we done on Kourou, let's use it on the Admin Console

## ToDo 
- [x] Integrate Kepler Companion in the Admin Console
- [x] Make a "cookie banner" to permit users to disable it (@Njuelle I'll need your help on this) 
- [x] Add current environment name `console.kuzzle.io` or `next-console.kuzzle.io` in Kepler tags

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -21,8 +22,17 @@ Reflect.defineProperty(window, 'kuzzle', {
 Vue.use(BootstrapVue)
 Vue.use(BootstrapVueIcons)
 
+const analytics = new KeplerCompanion()
 const router = createRoutes(Vue.prototype.$log)
 
+router.afterEach(async (to, _) => {
+  await analytics.track({
```
Comment (Aschen) [inline src/main.ts:29]: You should not await here but only empty catch to not degrade the UX

**Review by Aschen — APPROVED** [review]

File: src/routes/index.ts [hunk 2]
```diff
@@ -152,5 +154,24 @@ export default function createRoutes(log) {
...
 
+  const analytics = new KeplerCompanion()
+  router.afterEach(async (to, _) => {
+    const shouldAddTelemetry =
+      telemetryCookies.get() === null || telemetryCookies.get() === 'false'
+        ? false
+        : true
+
+    if (shouldAddTelemetry) {
+      await analytics.add({
```
Comment (Aschen) [inline src/routes/index.ts:165]: Can you add a catch to this promise so it never prints error to the console?

**Review by Aschen — CHANGES_REQUESTED** [review]

**Review by Njuelle — APPROVED** [review]

**Comment by Njuelle** [comment]
the tests fail because the new banner covers elements of the DOM and prevents the click on these elements. IMHO the best way to fix this would be to disable the banner and the telemetry during the tests

**Review by Aschen — APPROVED** [review]

**Comment by Aschen** [comment]
LGTM :+1:

**Pull request merged**